cmd/observe/flows.go

```diff
@@ -349,23 +395,36 @@ more.`,
 	formattingFlags.StringVarP(
 		&formattingOpts.output, "output", "o", "compact",
 		`Specify the output format, one of:
- compact:  Compact output
- dict:     Each flow is shown as KEY:VALUE pair
- json:     JSON encoding
- jsonpb:   Output each GetFlowResponse according to proto3's JSON mapping
- table:    Tab-aligned columns
+  compact:  Compact output
+  dict:     Each flow is shown as KEY:VALUE pair
+  json:     JSON encoding
+  jsonpb:   Output each GetFlowResponse according to proto3's JSON mapping
+  table:    Tab-aligned columns
 `)
 	formattingFlags.BoolVarP(&formattingOpts.nodeName, "print-node-name", "", false, "Print node name in output")
 	formattingFlags.StringVar(
 		&formattingOpts.timeFormat, "time-format", "StampMilli",
 		fmt.Sprintf(`Specify the time format for printing. This option does not apply to the json and jsonpb output type. One of:
```

Comment:
I don't think the formats introduced by this PR make a lot of sense for output. On the other hand, it's probably less confusing that way because of the symmetry with `--since` and `--until`, so no objection here.

Reply:
I agree. My take is that it would be more code to disallow this format for output and as it also doesn't hurt to have it, I thought it's fine.